### ISSUE TITLE
Title, Confirm and Cancel in Dark mode is not visible

### ISSUE THREAD

**Issue by theDevSoham** [henninghall/react-native-date-picker#720]
**Describe the bug**
Only tested on android: The dark mode text for the calendar is not changing color with theme change.

**Expected behavior**
To be set to white when in dark mode or preferably should take a customizalbe React.Fragment instead of a string

**To Reproduce**
Add example code that reproduces the behavior. 
```javascript
export default class App extends Component {

  state = { date: new Date() }

  render = () =>
    <DatePicker
                modal
                open={true}
                date={state }
                onConfirm={(d) => {
                  console.log("Date changed: ", d)
                }}
                onCancel={() => {
                  console.log("Cancelled: ")
                }}
                theme="dark"
                title="Select Date"
              />

}
```

**Smartphone (please complete the following information):**
 - OS: [Android]
 - React Native version [e.g. 0.67.3]
 - react-native-date-picker version [e.g. 4.3.3]

- For Light mode:

![image](https://github.com/henninghall/react-native-date-picker/assets/79685489/1fdae0ac-f8ec-4769-b703-547499e00453)

 
- For Dark mode:

![image](https://github.com/henninghall/react-native-date-picker/assets/79685489/359da2a1-862e-4cbe-ae33-7b070e046722)

**Comment by theDevSoham** [comment]
@henninghall please check

**Comment by brandon-austin-lark** [comment]
Same issue here

**Comment by cyrilmarceau** [comment]
Same issue but only on google device. I have tested on a samsung s10e and problem will not appear

**Comment by pranjal-easy-peasy** [comment]
We've had to work around this by using `theme="light"` but it would be great to get this fixed.

**Comment by theDevSoham** [comment]
`theme="light"` actually works fine...there's only some issue with dark mode heading and bottom buttons. It really needs some fix.

**Comment by z1haze** [comment]
This same issue exists for me with the date picker fields. This can be seen in #734

**Comment by mkhoussid** [comment]
Is there a planned fix or we might as well fork this repo and do it ourselves?

**Comment by henninghall** [comment]
PRs are welcome!

**Comment by vinamrahalgar** [comment]
cancelText does it work for iOS as well?

**Comment by statico** [comment]
Thank you!

**Comment by henninghall** [comment]
Fixed in v5.0.6

**Comment by ericpoulinnz** [comment]
Still broken on my Google Pixel 6 using Rn 74.5
